src/roaring.c

```diff
@@ -2005,20 +2013,26 @@ static void inplace_fully_flip_container(roaring_array_t *x1_arr, uint16_t hb) {
 roaring_bitmap_t *roaring_bitmap_flip(const roaring_bitmap_t *x1,
                                       uint64_t range_start,
                                       uint64_t range_end) {
-    if (range_start >= range_end) {
+    if (range_start >= range_end || range_start > (uint64_t)UINT32_MAX + 1)
+        return roaring_bitmap_copy(x1);
```

Comment:
Nit/style preference: most of the codebase uses parentheses even for single-statement if expressions. Same elsewhere in this PR.

Reply:
@SLieve Maybe you mean curly brackets? As in `if(condition) { dothis }`  as opposed to if(condition)  dothis `.

Reply:
Err yes, my mistake!